javaScript-week2/flightBooking.js

```diff
@@ -0,0 +1,21 @@
+function getFullname(firstname, surname, useFormalName, isFemale) {
+  if (firstname === "" && surname === "") {
+    return alert("Please enter your name");
+  } else if (useFormalName) {
+    if (isFemale) {
+      return `Madam ${firstname} ${surname}`;
+    } else {
+      return `Lord ${firstname} ${surname}`;
+    }
+  } else {
+    return `${firstname} ${surname}`;
+  }
+}
+
+const fullName1 = "Dennis";
+const fullName2 = "Agyekum";
+const useFormalName = true;
+const female = false;
+const printOut = getFullname(fullName1, fullName2, useFormalName, female);
```

Comment:
The conditional logic doesn't work very well for one case:
```
getFullName("Angelina", "Jolie", true);
```
It will return `Lord Angelina Jolie`. It will be better to only the full name, in case the argument for `isFemale` is undefined.